Session.php

```diff
@@ -61,6 +61,9 @@ public function init()
     {
         parent::init();
         $this->db = Instance::ensure($this->db, Connection::className());
+        $this->db
+            ->getCollection($this->sessionCollection)
+            ->createIndex('expire', ['expireAfterSeconds' => 0]);
```

Comment:
This is not appropriate.
`createIndex()` will be called at each application request process.